pkgs/development/r-modules/README.md

```diff
@@ -79,6 +79,30 @@ environment, see `rstudioWrapper`, which functions similarly to
 Then like above, `nix-env -f "<nixpkgs>" -iA rstudioEnv` will install
 this into your user profile.
 
+Alternatively, if you would rather not polute your environement or want
```

Comment:
I think the term "pollute" is inappropriate here. You don't pollute an environment by installing packages into it.

Reply:
Hmm... agreed. Not the best wording. I was trying to refer to the fact with the current example you need to go and modify some configuration files, most likely your config.nix and you may want different things for different projects. I'll update that paragraph to make it clear and use less opinionated terms.